docs/Installation.md

```diff
@@ -17,7 +20,7 @@ _**NOTE:**_
 
         sudo apt-get update
         sudo apt-get install -y python3 python3-pip gcc git python3-dev libssl-dev libffi-dev cron python3-lxml apache2 libapache2-mod-wsgi-py3
-        sudo pip3 install 'cffi==1.11.5' 'cryptography==1.4' Flask launchpadlib simplejson requests pytest
+        sudo pip3 install 'cffi==1.11.5' 'cryptography==1.4'  
```

Comment:
I haven't tested this, are these packages that you removed from this installation not required?

Reply:
Sorry about this, serious blunder from me. The first two packages, cffi and cryptography are the ones which aren't required. However, I didn't want to remove them yet. Anyways, the last 4 packages are very much required. I'll fix this